### ISSUE TITLE
Weird Texas totals from March 9-23 

### ISSUE THREAD

**Issue by chrisalcantara** [BuzzFeedNews/zika-data#8]
I'm pulling the same information from the CDC and came across the issue of the center saying Texas had 19 travel-associated cases on March 9, then 34 cases on March 16 and then down to 23 on March 23. 

I asked the CDC about the arch, and they responded Texas had double counted. Myself and another reporter have asked the Texas Department of Health about this and what they reported to the CDC. 

Thought I'd bring this up because 34 cases in Texas for March 16 does not seem correct.

**Comment by jsvine** [comment]
Thanks, Chris. I agree: 34 cases in Texas for March 16 seems off. Because the goal with the historical files is simply to document what the CDC was _reporting_ at the time, I'm going to keep these files as-is. But many thanks for flagging this particular anomaly! Great to know about it.

**Comment by chrisalcantara** [comment]
Of course. I can provide an update if Texas responds in case anyone finds the same anomaly and looks at this issue. Hopefully, they can provide the correct number, so the live-tracking data can be accurate. I understand keeping things as-is is also important.

**Comment by jsvine** [comment]
That'd be fantastic. Thanks, Chris!
